data-protocols/ids/ids-api-transfer/build.gradle.kts

```diff
@@ -28,6 +29,9 @@ dependencies {
 
     implementation("jakarta.ws.rs:jakarta.ws.rs-api:${rsApi}")
 
+    testImplementation("org.glassfish.jersey.core:jersey-common:${jerseyVersion}")
+    testImplementation("com.github.javafaker:javafaker:1.0.2")
```

Comment:
Since it's a test dependency that could be used by any module it could be moved to the root `build.gradle.kts`

Reply:
Thanks for the suggestion. I moved the java faker dependency to the root